### PR TITLE
docs: Update Exp and BG for FWC

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary

- [x] Add FWC to Exp page
- [x] Add FWC and Firetests to beginner guide

### Location

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

File: docs/pilots-corner/beginner-guide/starting-the-aircraft.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Aircraft is in a cold and dark state at a gate.
 
 ## Cockpit Preparation
 
-To start off, we will begin by doing a few pre-checks in all areas of the cockpit to ensure the aircraft is in an normal position before turning on the batteries.
+To start off, we will begin by doing a few pre-checks in all areas of the cockpit to ensure the aircraft is in an
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/starting-the-aircraft.md]:
```suggestion
To start off, we will begin by doing a few pre-checks in all areas of the cockpit to ensure the aircraft is in a 
```

File: docs/pilots-corner/beginner-guide/starting-the-aircraft.md [hunk 2]
```diff
@@ -45,7 +46,8 @@ We then direct our attention to the **main panel** and make sure the following i
 
 - [Gear Lever](../a32nx-briefing/flight-deck/front/autobrake-gear.md) is down
 
-Then we look up at the **overhead panel**. Make sure the following are in the set position before moving on to starting the aircraft:
+Then we look up at the **overhead panel**. Make sure the following are in the set position befor  moving on to
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/starting-the-aircraft.md]:
```suggestion
Then we look up at the **overhead panel**. Make sure the following are in the set position before moving on to 
```

File: docs/pilots-corner/beginner-guide/starting-the-aircraft.md [hunk 3]
```diff
@@ -168,15 +205,23 @@ Managing the temperatures of the cabin is necessary as well. Simply rotate the 3
...
 
 Just below the hydraulic panel, we find the **fuel panel**. Turn on all 6 fuel pumps until the white `OFF` light is no longer visible.
 
 ![Fuel Pumps](../assets/beginner-guide/starting-aircraft/fuelpumps.png "Fuel Pumps"){loading=lazy}
 
+### ENG Fire Test
+
+![Fire Panel](../assets/beginner-guide/starting-aircraft/fire-panel.png "Fire Panel")
+
+To perform an Engine fire test, hold down the `TEST` button for each of them for 5 seconds minimum to ensure
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/starting-the-aircraft.md]:
```suggestion
To perform an engine fire test, hold down the `TEST` button for each of them for 5 seconds minimum to ensure 
```
or "Engine Fire Test" -- unsure of your use case for capital E here.

**Review by Valastiri — COMMENTED** [review]
minor typos otherwise 👍

**Review by beheh — APPROVED** [review]
Thanks for putting this together!

**Pull request merged**